src/commands/guild/GuildDailyCommand.js

```diff
@@ -168,7 +168,7 @@ const GuildDailyCommand = async (language, message, args, forcedReward) => {
 
 	if (rewardType === REWARD_TYPES.HOSPITAL) {
 		for (const i in members) {
-			await members[i].Player.fastForward(Math.round(guild.level / 20))
+			require("../../core/Maps.js").advanceTime(members[i].Player, Math.round(guild.level / 20) * 60);
```

Comment:
@eagles666 Évite de `require("../../core/Maps.js")` comme ça (tu vas import "Maps" pour chaque itération de la boucle).
Favorise plutôt un seul `require("../../core/Maps.js")` en début de fichier.